tests/containers/agency/AgencyFooterContainer-test.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * AgencyFooterContainer-test.jsx
+ * Created by Emily Gullo 09/22/2017
+ */
+
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+
+import { AgencyFooterContainer } from
+    'containers/agency/AgencyFooterContainer';
+import Router from 'containers/router/Router';
+
+import { mockLoad } from './mockAgency';
+
+
+const routerPushSpy = sinon.spy(Router.history, 'push');
+const loadAgencySpy = sinon.spy(AgencyFooterContainer.prototype, 'loadAgency');
+
+// force Jest to use native Node promises
+// see: https://facebook.github.io/jest/docs/troubleshooting.html#unresolved-promises
+global.Promise = require.requireActual('promise');
+
+describe('AgencyFooterContainer', () => {
+    describe('loadAgency', () => {
+        it('should set the state of the loaded agency', async () => {
+            const container = mount(<AgencyFooterContainer
+                clearAllFilters={jest.fn()}
+                updateSelectedAwardingAgencies={jest.fn()} />);
+
+            container.instance().loadAgency('123');
```

Comment:
`loadAgency` calls `AgencyHelper`, which is not mocked. As a result, this makes a real API call. Unit tests are not integration tests and should have a scope restricted only to the target unit (`AgencyFooterContainer`) - we should assume that all the inputs are correct (they would be tested separately). Additionally, if the API data changes (such as the DB IDs change), the tests could fail even if the code itself is still valid.

Please make sure you are mocking the API calls. A good way to validate you are mocking data is to use clearly fake data (make up a fake agency) rather than basing the mock data in `mockAgency` on real API responses. Because the API will never return this fake data, you will know definitively if your test mocks are in fact mocks or if they are calling the real API.